Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1

```diff
@@ -4,7 +4,7 @@ Clear-Host
 [string] $salesforcePassword = $args[1]
 [string] $relativityVersion = $args[2]
 [Boolean] $sendSlackMessage =  $args[3]
-[string] $relativityVersionReleaseName = ''
+[string] $relativityVersionReleaseName = $args[4]
```

Comment:
Please remove _Clear-Host_ from the top of this script file. When running the Automation script, this line clears all the console output from the Automation script.